src/components/search/search.njk

```diff
@@ -1,35 +1,39 @@
-<h6>Search big</h6>
+<section aria-labelledby="search-big-header">
+  <h1 id="search-big-header">Search big</h1>
 
-<form class="usa-search usa-search--big">
-  <div role="search">
-    <label class="usa-sr-only" for="search-field-big">Search</label>
-    <input class="usa-input" id="search-field-big" type="search" name="search">
-    <button class="usa-button" type="submit">
-      <span class="usa-search__submit-text">Search</span>
-    </button>
-  </div>
-</form>
+  <form class="usa-search usa-search--big" aria-label="Search big" role="search">
```

Comment:
Seems like maybe an ambiguous use of `aria-label` — is there a better way to describe this element?

Reply:
I'll admit, I'm not the greatest at writing copy. Maybe `aria-label="Big variation of search"`?

Reply:
Removed, as this is unnecessary